public/reportissue.php

```diff
@@ -181,43 +222,40 @@ function displayCore() {
             <tr>
                 <td><label for="description">Description</label></td>
                 <td colspan="2">
-                    <textarea class="w-full forum" name="description" id="description"
-                              style="height:160px" rows="5" cols="61" placeholder="Describe your issue here..."
-                              required data-bind="textInput: description"><?= old('description') ?></textarea>
-                    <p data-bind="visible: descriptionIsNetworkProblem">Please do not use this tool for network issues. See <a href='https://docs.retroachievements.org/FAQ/#how-can-i-get-credit-for-an-achievement-i-earned-but-wasnt-awarded'>here</a> for instructions on how to request a manual unlock.</p>
-                    <p data-bind="visible: descriptionIsUnhelpful">Please be more specific with your issue&mdash;such as by adding specific reproduction steps or what you did before encountering it&mdash;instead of simply stating that it doesn't work. The more specific, the better.</p>
+                    <textarea
+                        class="w-full forum"
+                        name="description"
+                        id="description"
+                        style="height:160px"
+                        rows="5"
+                        cols="61"
+                        placeholder="Describe your issue here..."
```

Comment:
The cursor acts like the watermark is there, but it's not visible
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/82dea36f-6355-4044-90ee-c870ce53d4f0)
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0daf1dfb-0db4-4b10-99b9-7d45556a67cd)

Reply:
And typing doesn't clear it out either. I can click in the middle of the placeholder text and add some letters.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/1077aff6-2e78-4c33-b921-95578afb2065)

The submitted issue captures the behavior as whitespace:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9e27e7c0-2ba4-48fa-bd96-17101ea72b52)

Reply:
And it seems to interfere with the regex matching:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e0792b64-3896-4d55-8cba-a0eefb291fdc)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9461029d-0dae-4425-88de-6cae4ca079df)

Reply:
Resolved in latest. This turned out to be a rendering issue with Blade templates.

Note: The "no work" matching does cut off at 25 characters.